### DISCUSSION TITLE
"could not find `android` in `slint`"

### DISCUSSION THREAD

**Discussion by Apis48463** [slint-ui/slint#5187] [Q&A]
Hello.

I can't use slint::android even though i have backend-android-activity-05 feature enabled in my cargo.toml:

   [lib]
   crate-type = ["cdylib"]

   [dependencies]
   slint = { version = "1.5.1", features = ["backend-android-activity-05"] }

this is my error message:

error[E0433]: failed to resolve: could not find `android` in `slint`
   --> src\lib.rs:4:12
    |
4   |     slint::android::init(app).unwrap();
    |            ^^^^^^^ could not find `android` in `slint`
    |
note: found an item that was configured out
   --> C:\Users\janma\.cargo\registry\src\index.crates.io-6f17d22bba15001f\slint-1.5.1\lib.rs:322:9
    |
322 | pub mod android;
    |         ^^^^^^^

I also set crate type to cdylib and renamed my main.rs to lib.rs just like in documentation.
I have no idea why this is happening.

**Comment by ogoffart** [comment]
Are you compiling for android with the right --target?
What is the command line you are using to build?
